src/pages/monitor/MonitorPage.tsx

```diff
@@ -215,6 +218,7 @@ const MonitorPage = () => {
         params.row.state
           ? [
               <GridActionsCellItem
+                id="Inserire KO"
```

Comment:
@leleOFA suggerisco di cambiare in "KO-insert" e "OK-insert"

Reply:
fix it